src/spaces/graph.jl

```diff
@@ -127,3 +137,39 @@ function nearby_positions(
     filter!(i -> i != position, output)
 end
 
+#######################################################################################
+# Mutable graph functions
+#######################################################################################
+export rem_node!, add_node!, add_edge!
+
+"""
+    rem_node!(model::ABM{A, <: GraphSpace}, n::Int)
+Remove node (i.e. position) `n` from the model's graph. All agents in that node are killed.
+
+**Warning:** LightGraphs.jl (and thus Agents.jl) swaps the index of the last node with
+that of the one to be removed, while every other node remains as is. This means that
+when doing `rem_node!(n, model)` the last node becomes the `n`-th node while the previous
+`n`-th node (and all its edges and agents) are deleted.
+"""
+function rem_node!(model::ABM{<:AbstractAgent, <: GraphSpace}, n::Int)
+    for id ∈ copy(ids_in_position(n, model)); kill_agent!(model[id], model); end
+    V = nv(model)
+    success = LightGraphs.rem_vertex!(model.space.graph, n)
+    n ∉ 1:V && error("Node number exceeds amount of nodes in graph!")
```

Comment:
Lines 156 and 158 should be the first two lines of the function.

Isn't `n > V` faster than `n ∉ 1:V`?

Reply:
yes, by 1 boolean operation. `n \in range` reduces to:
```
in(x::Integer, r::AbstractUnitRange{<:Integer}) = (first(r) <= x) & (x <= last(r))
```